pkg/client/block/client_test.go

```diff
@@ -44,7 +44,7 @@ func TestBlockClient(t *testing.T) {
 	expectedEventBz, err := json.Marshal(expectedBlockEvent)
 	require.NoError(t, err)
 
-	eventsQueryClient := testeventsquery.NewAnyTimesEventsBytesEventsQueryClient(
+	eventsQueryClient := testeventsquery.NewAnyTimesClosableEventsQueryClient(
```

Comment:
What's the rationale for this? The pattern behind the naming thus far has been `New<num_times><main_mocked_method><interface_name>()`.